dev-packages/application-manager/src/generator/backend-generator.ts

```diff
@@ -35,6 +35,13 @@ const { BackendApplication, CliManager } = require('@theia/core/lib/node');
 const { backendApplicationModule } = require('@theia/core/lib/node/backend-application-module');
 const { messagingBackendModule } = require('@theia/core/lib/node/messaging/messaging-backend-module');
 const { loggerBackendModule } = require('@theia/core/lib/node/logger-backend-module');
+const { BackendApplicationConfigProvider } = require('@theia/core/lib/node/backend-application-config-provider');
+
+const applicationConfigProvider = BackendApplicationConfigProvider.get();
+const config = JSON.parse(\`
+${this.prettyStringify(this.pck.props.backend.config)}
+\`);
+applicationConfigProvider.initialize(config);
```

Comment:
Would it be possible to generate JSON object here directly without parsing at runtime?

Reply:
I did not find a better way, if you're aware of a better approach, let me know and I'll adjust it.

Reply:
I though `JSON.stringify` would do? 
```ts
applicationConfigProvider.set(${JSON.stringify(this.pck.props.backend.config)})
```

Reply:
I am checking it...